Library/Homebrew/sandbox.rb

```diff
@@ -8,6 +8,14 @@ def self.available?
     OS.mac? && File.executable?(SANDBOX_EXEC)
   end
 
+  # there're times sandbox is inconvenience
+  def self.auto_disable?
+    if (auto_disable = ARGV.interactive? || ARGV.debug?)
+      opoo "The sandbox cannot be used in debug or interactive mode."
```

Comment:
Still feels potentially weird to have a `?` method which is normally a boolean getter print a message without any arguments? Could maybe take an argument so it's only printed once?